client/src/scenes/Signup/index.jsx

```diff
@@ -109,113 +93,140 @@ class Signup extends React.Component {
+                          placeholder="Username"
+                          name="username"
+                          type="text"
+                          error={!username.valid}
+                          onChange={this.changeHandler}
+                          onBlur={this.validateHandler}
+                          required
+                          icon={{
+                              name: 'check',
+                              className: `icon-green ${username.value && username.valid ? '' : 'icon-hidden'}`
+                          }}
+                      />
+                      <Label className="signup-pointing-label" pointing>
+              Username can contain alphanumeric characters and single hyphens, cannot begin or end with a hyphen
+                      </Label>
+                  </Form.Field>
+                  <Form.Field required>
+                      <label htmlFor="email">Email</label>
+                      <Form.Input
+                          fluid
+                          placeholder="Email"
+                          name="email"
+                          type="email"
+                          error={!email.valid}
+                          onChange={this.changeHandler}
+                          onBlur={this.validateHandler}
+                          required
+                          icon={{
+                              name: 'check',
+                              className: `icon-green ${email.value && email.valid ? '' : 'icon-hidden'}`
+                          }}
+                      />
+                  </Form.Field>
+                  <Form.Field required>
+                      <label htmlFor="password">Password</label>
+                      <Form.Input
+                          fluid
+                          placeholder="Password"
+                          name="password"
+                          type="password"
+                          error={!password.valid}
+                          onChange={this.changeHandler}
+                          onBlur={this.validateHandler}
+                          required
+                          icon={{
+                              name: 'check',
+                              className: `icon-green ${password.value && password.valid ? '' : 'icon-hidden'}`
+                          }}
+                      />
+                      <Label className="signup-pointing-label" pointing>
+              Password should be at least 8 characters including a number and a lowercase letter
+                      </Label>
+                  </Form.Field>
+
+                  <Button type="submit" color="green" fluid size="large" disabled={!formValid}>
+            Sign Up for Depot
+                  </Button>
+                  <Message error content={signupError} />
+              </Segment>
+          </Form>
+      );
+  };
+
+  renderGoogleAuth = () => {
+      const serverLoginURL = `${process.env.REACT_APP_SERVER_URL}/auth/google`;
```

Comment:
```suggestion
      const serverLoginURL = '/auth/google';
```

Reply:
Cannot fix, added app.config.js for direct links